core/trino-server/pom.xml

```diff
@@ -53,6 +59,8 @@
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-enforcer-plugin</artifactId>
                 <configuration>
+                    <!-- Temporarily skip until archive is reduced in size -->
```

Comment:
Why is this triggering now and not before? This change shouldn't affect the size of the tarball

Reply:
oh.. I forgot to take it out .. it was triggering because I pulled things in twice with different folders .. will remove